airflow/contrib/operators/jenkins_job_trigger_operator.py

```diff
@@ -0,0 +1,147 @@
+# -*- coding: utf-8 -*-
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import time
+from airflow.exceptions import AirflowException
+from airflow.models import BaseOperator
+from airflow.utils.decorators import apply_defaults
+from airflow.contrib.hooks.jenkins_hook import JenkinsHook
+import jenkins
+import json
+from six.moves.urllib.request import Request
+from six.moves.urllib.parse import urlencode, quote_plus
+
+from airflow.utils.log.LoggingMixin import LoggingMixin
+log = LoggingMixin().logger
```

Comment:
Add the mixin to the class please. Python supports multiple inheritance

Reply:
Done

Reply:
Sorry baseoperator already uses the mixin. You can remove it from the definition. (My mistake)

Reply:
Fixed, I'll pushed that as soon as I find how to do lazy logging in Python ^^ (it was definitely not a long thing to fix...^^ )